package.json

```diff
@@ -60,26 +60,26 @@
     "zone.js": "0.14.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "17.0.3",
-    "@angular-eslint/builder": "17.1.0",
-    "@angular-eslint/eslint-plugin": "17.1.0",
-    "@angular-eslint/eslint-plugin-template": "17.1.0",
-    "@angular-eslint/schematics": "17.1.0",
-    "@angular-eslint/template-parser": "17.1.0",
-    "@angular/cli": "17.0.3",
-    "@angular/compiler-cli": "17.0.4",
-    "@angular/language-service": "17.0.4",
+    "@angular-devkit/build-angular": "18.0.2",
+    "@angular-eslint/builder": "18.0.0",
+    "@angular-eslint/eslint-plugin": "18.0.0",
+    "@angular-eslint/eslint-plugin-template": "18.0.0",
+    "@angular-eslint/schematics": "18.0.0",
+    "@angular-eslint/template-parser": "18.0.0",
+    "@angular/cli": "18.0.2",
+    "@angular/compiler-cli": "18.0.1",
+    "@angular/language-service": "18.0.1",
```

Comment:
any reason that some of the `@angular` deps are not all aligned to `18.0.1`? I see some `18.0.2` (or all aligned to the latest)?

Reply:
I believe there were no higher versions available last week for those dependencies. I'll check again and see if they are available now. Can also revert everything to 18.0.0 if you want to?

Reply:
Might be worth having a look again if there's anything new and aligned but if not, I think it's fine. Let me know

Reply:
I Just checked the latest version of all angular related dependencies. Some are on version 18.0.1, some on 18.0.2 and some have a newer version 18.0.3. So it feels a little artifical to align the dependencies.